snuba/web/views.py

```diff
@@ -229,7 +251,7 @@ def parse_request_body(http_request):
         try:
             return json.loads(http_request.data)
         except json.errors.JSONDecodeError as error:
```

Comment:
```suggestion
        except json.JSONDecodeError as error:
```

This should probably be updated to stay the same as the change above.